testsuite/gnat2goto/tests/attrib_size_object/test.out

```diff
@@ -0,0 +1,15 @@
+Standard_Output from gnat2goto object_size:
+----------At: Process_Declaration----------
+----------size clause not applied by the front-end----------
```

Comment:
Although the tests pass, this still has an unsupported feature so I don't think it should pass.

Reply:
Fixed